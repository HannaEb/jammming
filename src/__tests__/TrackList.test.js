import TrackList from '../Components/TrackList/TrackList.js';
import renderer from 'react-test-renderer';

jest.mock('../Components/Track/Track', () => 'Track');

describe('TrackList', () => {
    it('matches the snapshot', () => {
        const tracks = ['Track1', 'Track2', 'Track3']
        const tree = renderer.create(<TrackList tracks={tracks} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
