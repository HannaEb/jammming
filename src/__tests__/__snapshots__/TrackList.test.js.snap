// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrackList matches the snapshot 1`] = `
<div
  className="TrackList"
>
  <Track
    track="Track1"
  />
  <Track
    track="Track2"
  />
  <Track
    track="Track3"
  />
</div>
`;
